installer/ubuntu/bootstrap

```diff
@@ -54,6 +51,9 @@ else
     echo "Unable to patch debootstrap, moving on..." 1>&2
 fi
 
+# Patch debootstrap so that is does not create devices under /dev (issue #2387).
+sed -i -e 's/\(setup_devices () {\)/\1 return 0/' "$tmp/functions"
```

Comment:
Actually, the regex is missing checks for line beginning & end, so it could match another function.